pkg/systemd/services/timer-dialog.jsx

```diff
@@ -233,27 +248,56 @@ const CreateTimerDialogBody = ({ owner }) => {
                                         }}
                                         aria-label={_("Repeat")}>
                                 <FormSelectOption value="no" label={_("Don't repeat")} />
+                                <FormSelectOption value="minutely" label={_("Minutely")} />
                                 <FormSelectOption value="hourly" label={_("Hourly")} />
                                 <FormSelectOption value="daily" label={_("Daily")} />
                                 <FormSelectOption value="weekly" label={_("Weekly")} />
                                 <FormSelectOption value="monthly" label={_("Monthly")} />
                                 <FormSelectOption value="yearly" label={_("Yearly")} />
+                                <FormSelectOption value="custom" label={_("Custom")} />
                             </FormSelect>
                         </FormGroup>
                         {repeat == "no" &&
                         <FormGroup label={_("Run at")}>
                             <TimePicker className="create-timer-time-picker specific-no-repeat"
                                         menuAppendTo={() => document.body} time={specificTime} is24Hour onChange={setSpecificTime} />
                         </FormGroup>}
+                        {repeat == "custom" &&
+                        <FormGroup
+                            label={_("Custom")}
+                            validated={submitted && validateCustom ? "error" : "default"}
+                            helperTextInvalid={_("Invalid custom event")}>
+                            <Flex>
+                                <TextInput
+                                    id="custom"
+                                    value={custom}
+                                    onChange={setCustom}
+                                    validated={submitted && validateCustom ? "error" : "default"}
+                                />
+                                <HelperText>
+                                    <HelperTextItem>Use systemd oncalendar format.  Example *:0/5</HelperTextItem>
```

Comment:
This needs to be localized. `<HelperTextItem>{_("Use systemd oncalendar format.  Example *:0/5")}</HelperTextItem>`

Reply:
oops.  missed that one.  ill get this cleaned up with Garrets suggestions.